WBooks/LibraryViewController.swift

```diff
@@ -10,42 +10,76 @@ import Foundation
 import Networking
 import ReactiveSwift
 import UIKit
+import WolmoCore
 
 final class LibraryViewController: UITableViewController {
     
     private let _bookRepo = NetworkingBootstrapper.shared.createWBooksRepository()
     private var _bookArray = [Book]()
+    private var _AllBooksRead = false
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        navigationItem.title = NSLocalizedString("NAVIGATION_BAR_TITLE_LIBRARY", comment: "")
+        navigationItem.title = "NAVIGATION_BAR_TITLE_LIBRARY".localized()
+        tableView.register(UINib(nibName: "BookCell", bundle: nil), forCellReuseIdentifier: "BookCell")
+        setTableBackground()
+        loadBooks()
         
-        _bookRepo.fetchBooks().observe(on: UIScheduler()).startWithResult{
-            switch $0 {
-            case .success(let books):
-                self._bookArray = books
-                self.tableView.reloadData()
-            case .failure(let error):  print("\(error)")
+    }
+    
+    func loadBooks(){
+        if _AllBooksRead == false {
+            _bookRepo.fetchBooks().observe(on: UIScheduler()).startWithResult{
```

Comment:
your should add `[unowned self]` after `startWithResult { ` if you are using `self` inside the block.
Check this https://krakendev.io/blog/weak-and-unowned-references-in-swift